### DISCUSSION TITLE
Should view extend a table's masked policy ?

### DISCUSSION THREAD

**Discussion by Karsonnel** [apache/kyuubi#5074] [General]
For example, i create a table such like table_a with a column_a masked, then i create a persist view with query: create view view_a as select * from table_a, i find that the column column_a in view_a is masked. Is this behavior expected?

**Comment by yaooqinn** [comment]
Literally, a masking policy is applied to the column level, which here refers to the view. But on second thought, if a user has permission to create views, does this mean he/she can use views to bypass masking?

I don't have a clear view of this question. Cc @bowenliang123, any thoughts?
